predix/admin/app.py

```diff
@@ -75,20 +68,20 @@ def lock_to_org_space(self):
         self.add_env_var('PREDIX_SPACE_NAME', self.space.name)
         self.write_manifest()
 
-    def create_uaa(self, admin_secret):
+    def create_uaa(self, admin_secret, **kwargs):
         """
         Creates an instance of UAA Service.
 
         :param admin_secret: The secret password for administering the service
             such as adding clients and users.
         """
-        uaa = predix.admin.uaa.UserAccountAuthentication()
+        uaa = predix.admin.uaa.UserAccountAuthentication(**kwargs)
         if not uaa.exists():
-            uaa.create(admin_secret)
+            uaa.create(admin_secret, **kwargs)
```

Comment:
This line should not be changed. `uaa.create()` does not accept any keyword argument. Creating an UAA with an argument such as `name` is impossible and raises the following error:
`TypeError: create() got an unexpected keyword argument 'name' `

Reply:
+1

Thanks!  I missed a commit on predix/admin/uaa.py.  Running through some more new integration tests and a few more files around eventhub still to be merged, but if you see anything else don't hesitate to bring attention to it.